content/en/community/hackathons/2023-09-aveiro/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Aveiro 2023
+date: 2023-06-26T09:30:00+02:00
+draft: false
+collapsible: true
+description: Aveiro Unikraft Hackathon, September 15-16, 2023
+image: /assets/imgs/2023-09-hack-aveiro.png
+---
+
+## Aveiro Unikraft Hackathon
+
+Unikraft, [University POLITEHNICA of Bucharest](https://upb.ro/en), [University of Aveiro](https://www.ua.pt/), [Grupo de Linux da Universidade de Aveiro (GLUA)](https://glua.ua.pt/) and [Associação Nacional para o Software Livre (ANSOL)](https://ansol.org/) come together to organize the **Aveiro Unikraft Hackathon** to be held on Friday and Saturday, September 15-16, 2023, as part of [Festa do Software Livre 2023](https://festa2023.softwarelivre.eu/).
+
+The hackathon will take place as an in-person event at [DETI - Departamento de Electrónica, Telecomunicações e Informática](https://www.ua.pt/en/deti) in University of Aveiro.
+The full address is: [Electronics, Telecommunication and Informatics Department, University of Aveiro, Campus Universitário de Santiago, 3810-193 Aveiro, Portugal](https://goo.gl/maps/RoQuGBKuiueGuwhL7).
+
+Support information and discussions will take place on [Discord](http://bit.ly/UnikraftDiscord) on the `#hack-aveiro23` channel.
+
+### Registration
+
+To take part in the hackathon please fill this [registration form](https://forms.gle/wQo3rc4gYTgY7tWS8) by **Sunday, September 3, 2023, 11pm WEST**.
+
+Please bring your own laptop.
+It's best if you have a native Linux installed on your laptop.
+Otherwise, please install [this virtual machine](https://drive.google.com/file/d/1u5DtN5kMPWxBU8UdBfnZ7DNRP2n6oiTy/view?usp=share_link).
```

Comment:
```suggestion
Otherwise, please install [this virtual machine](https://drive.google.com/file/d/16oBxfjFvu5mpf6DMb4-bsnyd5n2FqReo/view?usp=sharing).
```
Update this to the new VM